torch/csrc/jit/script/compiler.h

```diff
@@ -177,6 +177,22 @@ struct TORCH_API AnnotateValue : public SugaredValue {
   }
 };
 
+// matched against for special handling of getattr expressions
+struct TORCH_API GetAttrValue : SugaredValue {
+  GetAttrValue() = default;
+  std::string kind() const override {
+    return "getattr";
+  }
+};
+
+// matched against for special handling of isinstance expressions
+struct TORCH_API IsInstValue : SugaredValue {
```

Comment:
easily confused with Instruction or something, can you spell out the whole thing `IsinstanceValue`